src/elements/swapLimit/SwapLimit.tsx

```diff
@@ -112,7 +118,7 @@ export const SwapLimit = ({
         setFromAmountUsd(usdAmount);
       }
     },
-    [fromToken]
+    [fromToken,setFromAmount,setFromAmountUsd]
```

Comment:
please run prettier

Reply:
done .

Reply:
![image](https://user-images.githubusercontent.com/9465239/143003505-ee104f9c-5026-4cc9-a2e8-b280fed21869.png)
this issue seems to be happening after i merged with main .this only happens when you select a receive token and then swap from market to limit . if its in 'select a token' mode its fine

Reply:
Thanks, it was opened today - #437

Reply:
shall i fix that in this ticket . i figured the issue . or should i create a seperate branch for it ?